internal/bminventory/inventory.go

```diff
@@ -5651,7 +5651,7 @@ func (b *bareMetalInventory) V2RegisterHost(ctx context.Context, params installe
 
 	if err = b.customizeHost(host); err != nil {
 		// TODO Need event for infra-env instead of cluster
```

Comment:
can the comment be removed or do we plan to remove it after persisting the infraenv_id of the event to the db ?

Reply:
Not sure. @filanov?